app/cli/cmd/root.go

```diff
@@ -300,7 +306,7 @@ func initConfigFile() {
 	}
 
 	// If no config file was passed as a flag we use the default one
-	configPath := filepath.Join(xdg.ConfigHome, appName)
+	configPath := plugins.GetConfigDir()
```

Comment:
it's strange that this generic config directory now it's in the plugins package, also hiding the `appName` property which happens to be relevant at the root level